README.md

```diff
@@ -20,7 +22,7 @@ conda create -n jbook-env python=3.10
 conda activate jbook-env
 ```
 
-Install packages:
+[Install the Jupyter Book Package](https://jupyterbook.org/en/stable/start/overview.html#install-jupyter-book), for example via pip:
```

Comment:
link to jupyter book installation guide, for people to know about how to use conda (if they like that kind of thing)